packages/office-ui-fabric-react/src/components/ExtendedPicker/examples/PeopleExampleData.ts

```diff
@@ -1,8 +1,8 @@
 // If this file is moved or split, the scripts for building codepen examples will likely need to be updated.
 
-import { PersonaPresence } from '../../../Persona';
-import { IExtendedPersonaProps } from '../../../SelectedItemsList';
-import { TestImages } from '../../../common/TestImages';
+import { PersonaPresence } from 'office-ui-fabric-react/lib/Persona';
+import { IExtendedPersonaProps } from 'office-ui-fabric-react/lib/SelectedItemsList';
```

Comment:
@JasonGore this is what I meant by my comment above. This module is exported from OUFR and we import something from within the same package. But unsure why `PersonaPresence` from the above line didn't end up in the same position. Perhaps some bug in the `api-extractor`